addon/components/o-s-s/attributes-panel.ts

```diff
@@ -25,9 +26,18 @@ export default class OSSAttributesPanel extends Component<OSSAttributesPanelArgs
     assert('[component][OSS::AttributesPanel] The @onSave parameter is mandatory', typeof args.onSave === 'function');
   }
 
+  get editIcon(): string {
+    return this.args.customEditIcon ?? 'fa-pen';
+  }
+
   @action
   toggleMode(): void {
     this.modeSelected = this.modeSelected === 'view' ? 'edition' : 'view';
+  }
+
+  @action
+  onEdit(): void {
```

Comment:
what about an event.stopPropagation even though it's not really useful now ? but if one day `args.onEdit` opens a modal, we will be happy to have it ;)